command/ca/token.go

```diff
@@ -513,20 +526,29 @@ func provisionerPrompt(ctx *cli.Context, provisioners provisioner.List) (provisi
 	}
 
 	if len(provisioners) == 1 {
-		var id, name string
+		var name, value string
 		switch p := provisioners[0].(type) {
 		case *provisioner.JWK:
-			name = p.Name
-			id = p.Key.KeyID
+			name = "Key ID"
+			value = p.Key.KeyID + " (" + p.Name + ")"
 		case *provisioner.OIDC:
-			name = p.Name
-			id = p.ClientID
+			name = "Client ID"
+			value = p.ClientID + " (" + p.Name + ")"
+		case *provisioner.GCP:
+			name = "Provisioner"
```

Comment:
Not sure Provisioner is the most clear name here. maybe Cloud Identity?

Reply:
That `name` causes that the CLI shows something like when there's only one:
```
✔ Provisioner: My Provisioner Name (GCP)
```
I think is the right name, because the user should know the concept of provisioners.

In fact, I would prefer to unify everything to something like:
```
✔Provisioner: Name (TYPE) [(other info)]
```
So it can look like:
```
✔Provisioner: mariano@smallstep.com (JWK) (kid: asdfasdfasdfasfasdf)
or 
✔Provisioner: Google (OIDC) (client: asdfasdfasdfasdfasfd)
```
or something similar

Reply:
What's confusing (to me) is that the `Name` attribute for JWK and OIDC is more specific (in that same block), but the one for cloud identities is just `Provisioner`. They are all provisioners, so to me that `Name` should be more specific (e.g. type of Provisioner) if you're going to do that for JWK and OIDC.